src/OrchardCore/OrchardCore.DisplayManagement.Liquid/LiquidViewTemplate.cs

```diff
@@ -117,6 +117,8 @@ public ShapeAccessor() : base(_getter)
                         return shape.Attributes;
                     case "Items":
                         return shape.Items;
+                    case "Metadata":
```

Comment:
Yes, all props defined by the `IShape` interface

For the switch i would say to use the new c# pattern, but not so good for the `default` if not a simple expression, but maybe worth to do it ;)

                return n switch
                {
                    nameof(IShape.Id) => shape.Id,
                    nameof(IShape.TagName) => shape.TagName,
                    ...
                    _ => (Func<object>)(() =>
                    {
                        if (shape.Properties.TryGetValue(n, out var result))
                        {
                            return result;
                        }
                        ...
                    })